less/lib/Dropdown.less

```diff
@@ -79,6 +79,49 @@
   left: auto;
   right: 0;
 }
+.Dropdown-menu--below {
+    padding-left: 10px !important;
+    padding-top: 10px !important;
+    bottom: inherit !important;
+    position: absolute !important;
+    top: 100% !important;
+    overflow: auto;
+    max-height: 45vh !important;
+    min-width: 189px !important;
```

Comment:
I must say that all these `!important` are a bad sign, too. It means we have too specific selectors somewhere else.